src/app/pages/questions/services/finish-task.service.ts

```diff
@@ -21,8 +21,9 @@ export class FinishTaskService {
 
   updateTaskToComplete(task): Promise<any> {
     return Promise.all([
-      this.schedule.updateTaskToComplete(task),
-      this.schedule.updateTaskToReportedCompletion(task),
+      this.schedule
+        .updateTaskToComplete(task)
```

Comment:
What happens if it fails to update completion? I think we need retry capability here so the app keeps retrying to update completion until successful.

Reply:
oh i see it is there in the splash page. great!